Dockerfile

```diff
@@ -76,27 +183,36 @@ ENV MSPASS_HOME /mspass
 # Add setup.py to install python components
 ADD setup.py /mspass/setup.py
 ADD python /mspass/python
-RUN pip3 install /mspass -v
+RUN pip3 install /mspass -v \
+	&& docker-clean
 
 # Install Jupyter notebook
-RUN pip3 --no-cache-dir install jedi==0.17.2 notebook==6.2.0
+RUN pip3 --no-cache-dir install jedi==0.17.2 notebook==6.2.0 && docker-clean
```

Comment:
The new base container here has jupyterlab built-in, so the notebook is no longer needed. This is also true to the other container below with jupyterlab.